elastic-agent-client.proto

```diff
@@ -9,86 +9,87 @@ package proto;
 option cc_enable_arenas = true;
 option go_package = "pkg/proto;proto";
 
-// Connection information sent to the application on startup so it knows how to connected back to the Elastic Agent.
-//
-// This is normally sent through stdin and should never be sent across a network un-encrypted.
-message ConnInfo {
-    // GRPC connection address.
-    string addr = 1;
-    // Server name to use when connecting over TLS.
-    string server_name = 2;
-    // Token that the application should send as the unique identifier when connecting over the GRPC.
-    string token = 3;
-    // CA certificate.
-    bytes ca_cert = 4;
-    // Peer certificate.
-    bytes peer_cert = 5;
-    // Peer private key.
-    bytes peer_key = 6;
+// Empty response message.
+message EmptyResponse {}
```

Comment:
Is it worth using the `google.protobuf.Empty` well known type instead? https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Empty

It should be built into the proto compiler so it doesn't add a dependency.